src/docs/Introduction.stories.mdx

```diff
@@ -0,0 +1,38 @@
+import { Meta, Story } from '@storybook/addon-docs';
+
+<Meta title="Introduction"/>
+
+<center>
+<img src="/moja-global-icon.jpg" alt="logo" height="225" width="390"/>
+</center>
+
+#
```

Comment:
What's this for?

Reply:
This is to add a slight gap between the image and the title, since I was unable to do so in any other way. Let me know if there's any replacement to that?